packages/ui-toolkit/src/form/select.js

```diff
@@ -1,10 +1,38 @@
+import styled from 'styled-components';
+import React from 'react';
+import remcalc from 'remcalc';
 import Baseline from '../baseline';
 import BaseInput, { Stylable } from './base/input';
-import React from 'react';
 
 const Select = Baseline(BaseInput(Stylable('select')));
 
+const SelectWrapper = styled.div`
+  position: relative;
+  display: inline-flex;
+  min-width: ${remcalc(200)};
```

Comment:
suggestion: allow for a fluid select.

```diff
diff --git a/packages/ui-toolkit/src/form/select.js b/packages/ui-toolkit/src/form/select.js
index 9fd3f454..51e34f91 100644
--- a/packages/ui-toolkit/src/form/select.js
+++ b/packages/ui-toolkit/src/form/select.js
@@ -1,6 +1,8 @@
 import styled from 'styled-components';
 import React from 'react';
+import { isNot } from 'styled-is';
 import remcalc from 'remcalc';
+
 import Baseline from '../baseline';
 import BaseInput, { Stylable } from './base/input';
 
@@ -9,30 +11,32 @@ const Select = Baseline(BaseInput(Stylable('select')));
 const SelectWrapper = styled.div`
   position: relative;
   display: inline-flex;
-  min-width: ${remcalc(200)};
+
+  ${isNot('fluid')`
+    min-width: ${remcalc(200)};
+  `};
 
   &:after {
     content: '';
-    width: 10px;
-    height: 10px;
+    width: ${remcalc(10)};;
+    height: ${remcalc(10)};
     background: url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iOSIgaGVpZ2h0PSI2IiB2aWV3Qm94PSIwIDAgOSA2IiB4bWxucz0iaHR0cDo
 vL3d3dy53My5vcmcvMjAwMC9zdmciPjx0aXRsZT5hcnJvdzogcmlnaHQ8L3RpdGxlPjxwYXRoIG
 Q9Ik05IDEuMzg2TDcuNjQ4IDAgNC41IDMuMjI4IDEuMzUyIDAgMCAxLjM4NiA0LjUgNnoiIGZpb
-Gw9IiM0OTQ5NDkiIGZpbGwtcnVsZT0iZXZlbm9kZCIvPjwvc3ZnPg==')
-      center center no-repeat;
-      display: block;
-      position: absolute;
-      top: 50%;
-      transform: translateY(-50%);
-      right: ${remcalc(12)};
+Gw9IiM0OTQ5NDkiIGZpbGwtcnVsZT0iZXZlbm9kZCIvPjwvc3ZnPg==') center center no-repeat;
+    display: block;
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    right: ${remcalc(12)};
   }
 `;
 
 /**
  * @example ./usage-select.md
  */
-export default ({ children, ...rest }) => (
-  <SelectWrapper>
+export default ({ children, fluid, ...rest }) => (
+  <SelectWrapper fluid={fluid}>
     <Select {...rest}>{children}</Select>
   </SelectWrapper>
 );
```

Be aware, this requires `styled-is@1.0.15`: 

```diff
diff --git a/packages/joyent-boilerplate/package.json b/packages/joyent-boilerplate/package.json
index 26a4e869..61184998 100644
--- a/packages/joyent-boilerplate/package.json
+++ b/packages/joyent-boilerplate/package.json
@@ -33,7 +33,7 @@
     "redux-form": "^7.0.3",
     "remcalc": "^1.0.8",
     "styled-components": "^2.1.2",
-    "styled-is": "^1.0.14"
+    "styled-is": "^1.0.15"
   },
   "devDependencies": {
     "babel-plugin-inline-react-svg": "^0.4.0",
```

```diff
diff --git a/packages/ui-toolkit/package.json b/packages/ui-toolkit/package.json
index fdfcdf3b..71571b9a 100644
--- a/packages/ui-toolkit/package.json
+++ b/packages/ui-toolkit/package.json
@@ -61,7 +61,7 @@
     "remcalc": "^1.0.8",
     "rnd-id": "^1.0.8",
     "styled-components": "^2.1.2",
-    "styled-is": "^1.0.14",
+    "styled-is": "^1.0.15",
     "svg-inline-loader": "^0.8.0",
     "unitcalc": "^1.1.0",
     "url-loader": "^0.6.2"
```

```diff
diff --git a/packages/my-joyent/package.json b/packages/my-joyent/package.json
index 0cfbf9f8..77f3ee2c 100644
--- a/packages/my-joyent/package.json
+++ b/packages/my-joyent/package.json
@@ -34,7 +34,7 @@
     "redux-form": "^7.0.3",
     "remcalc": "^1.0.8",
     "styled-components": "^2.1.2",
-    "styled-is": "^1.0.14",
+    "styled-is": "^1.0.15",
     "unitcalc": "^1.1.0"
   },
   "devDependencies": {
```